.env.example

```diff
@@ -8,4 +8,5 @@ DEV_RPC=
 GNOSIS_RPC=
 ETHEREUM_RPC=
 OPTIMISM_RPC=
-BASE_RPC=
\ No newline at end of file
+BASE_RPC=
+MODE_RPC=
```

Comment:
do we have mode rpc gateway yet @Tanya-atatakai?

Reply:
afaik we don't and should use the public one